microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -45,33 +47,36 @@ Here are the main differences between the DevTools default user interface and Fo
 
 The **Activity Bar** is the most visible difference from the default user interface. Use it to open tools, help features, and settings.
 
-By default, the position of the **Activity Bar** adapts to the location of the DevTools window. If DevTools is undocked or docked to the bottom of the browser, the **Activity Bar** will be horizontal. If DevTools is docked to the side of the browser, the **Activity Bar** will be vertical, and tool names will not be visible. This enables quicker tool access when screen width is limited.
+By default, the position of the **Activity Bar** adapts to the location of the DevTools window:
+
+* If DevTools is undocked or docked to the bottom of the browser, the **Activity Bar** will be horizontal.
+* If DevTools is docked to the side of the browser, the **Activity Bar** will be vertical, and tool names will not be visible. This enables quicker tool access when screen width is limited.
 
-The **Activity Bar** top-aligned in Focus Mode, with DevTools docked:
+The **Activity Bar** left-aligned in Focus Mode, with DevTools docked to the side:
 
-![The Activity Bar top-aligned in Focus Mode, with DevTools docked](media/focus-mode-docked.png)
+![The Activity Bar left-aligned in Focus Mode, with DevTools docked](./focus-mode-images/docked.png)
 
 The **Activity Bar** top-aligned in Focus Mode, with DevTools popped out in a separate window:
 
-![The Activity Bar top-aligned in Focus Mode, with DevTools popped out in a separate window](media/focus-mode-undocked.png)
+![The Activity Bar top-aligned in Focus Mode, with DevTools popped out in a separate window](./focus-mode-images/undocked.png)
```

Comment:
show Elements tool, not Welcome tool, for familiar point of reference, for orientation